backend/src/main/java/com/google/sps/RemoveParticipantHelper.java

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import com.google.sps.datastore.ParticipantDatastore;
+import java.io.IOException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Helper class to execute doPost for RemoveParticipantServlet */
+public final class RemoveParticipantHelper {
+
+  private final ParticipantDatastore participantDatastore;
+  private final UsernameService usernameService;
+  private static final String RESPONSE = "Received remove request.";
```

Comment:
Why not have this hard coded into line 49?

Reply:
Since I took it out as a constant in the RemoveParticipantHelperTest, I decided to do the same thing in this class as well.